Tests/test_imagegrab.py

```diff
@@ -57,6 +57,11 @@ def test_grab_invalid_xdisplay(self) -> None:
             ImageGrab.grab(xdisplay="error.test:0.0")
         assert str(e.value).startswith("X connection failed")
 
+    @pytest.mark.skipif(sys.platform != "win32", reason="Windows only")
+    def test_grab_invalid_handle(self) -> None:
+        with pytest.raises(OSError):
+            ImageGrab.grab(handle=-1)
```

Comment:
```suggestion
        with pytest.raises(OSError):
            ImageGrab.grab(handle=-1)
        with pytest.raises(OSError):
            ImageGrab.grab(handle=0)
```
The value 0 is a special way to refer to the desktop. However, it cannot be captured in the same way as regular windows so it fails, but we can add a test for that.

Reply:
Thanks. I've added a commit.